### PR TITLE
Line of sight geoelement

### PR DESCRIPTION
**Pull request by tschie**
Shows how to add a line of sight between two moving graphics

### CODE REVIEW

File: src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <li>Create an <code>AnalysisOverlay</code> and add it to the <code>SceneView</code>'s analysis overlays collection.</li>
+    <li>Create a <code>LineOfSightGeoElement</code>, passing in observer and target <code>GeoElement</code>s (feautures or graphics). Add the line of sight to the analysis overlay's analyses collection</code>.</li>
+    <li>To get the target visibility when it changes, add a <code>TargetVisibilityChangedListener</code> to the line of sight. The changed event will give the <code>TargetVisibility</code>.</li>
+</ol>
+
+<h2>Relevant API</h2>
+
+<ul>
+    <li>AnalysisOverlay</li>
+    <li>LineOfSightGeoElement</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/README.md]: I see `GeoElementLineOfSight` but not the way that it is listed here.

File: src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/LineOfSightGeoElementSample.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+      // create waypoints around a block for the taxi to drive to
+      waypoints = Arrays.asList(
+          new Point(-73.984513, 40.748469, SpatialReferences.getWgs84()),
+          new Point(-73.985068, 40.747786, SpatialReferences.getWgs84()),
+          new Point(-73.983452, 40.747091, SpatialReferences.getWgs84()),
+          new Point(-73.982961, 40.747762, SpatialReferences.getWgs84())
+      );
+
+      // create a graphic of a taxi to be the target
+      String modelURI = new File("./samples-data/dolmus_3ds/dolmus.3ds").getAbsolutePath();
```
Comment (john0005) [inline src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/LineOfSightGeoElementSample.java:150]: May need to add this to samples data. I didn't see it appear after refreshing the data.

File: src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<p>A line of sight will display between a point on the Empire State Building (observer) and a taxi (target). The taxi will drive around a block and the line of sight should automatically update. The taxi will be highlighted when it is visibile. You can change the observer height with the slider to see how it affects the target's visibility.</p>
+
+<h2>How it works</h2>
+
+<p>To show a line of sight between two graphics:</p>
+
+<ol>
+    <li>Create an <code>AnalysisOverlay</code> and add it to the <code>SceneView</code>'s analysis overlays collection.</li>
+    <li>Create a <code>LineOfSightGeoElement</code>, passing in observer and target <code>GeoElement</code>s (feautures or graphics). Add the line of sight to the analysis overlay's analyses collection</code>.</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/analysis/line_of_sight_geoelement/README.md]: Need to change this name as well. `GeoElementLineOfSight`

**Review by john0005 — APPROVED** [review]
Minor changes. Looks good.

**Comment by tschie** [comment]
@john0005 I've made the updates and added the data. Merging.

**Pull request merged**